.github/workflows/build.yaml

```diff
@@ -11,21 +11,22 @@ jobs:
     runs-on: self-hosted
     timeout-minutes: 20
     container:
-      image: docker.io/mayadata/ms-buildenv:latest
+      image: docker.io/mayadata/ms-buildenv:gila
```

Comment:
guess you'll move this to latest before you merge?

Reply:
Yes :+1: